projects/demo/project.json

```
@@ -275,6 +277,37 @@
                     "includePaths": ["projects/core/styles"]
                 }
             }
+        },
+        "e2e": {
+            "builder": "@nrwl/workspace:run-commands",
+            "options": {
+                "parallel": false,
+                "commands": [
+                    "npx wait-on http://localhost:3333 --timeout 1000",
+                    "cypress open --browser chrome --project ./projects/demo-integrations"
+                ]
+            }
+        },
+        "e2e:headless": {
```

```suggestion
        "e2e-headless": {
```
(for uniformity of all commands' names)

---

fixed